### DISCUSSION TITLE
Setting up Python LSP Server with pyenv on macOS.

### DISCUSSION THREAD

**Discussion by hitblast** [helix-editor/helix#4330] [Q&A] [answered]
Hello! I'm a MacBook user and aiming to switch from Visual Studio Code to Helix. Most of my open-source projects are of Python so I'd like to setup `pylsp` for stuff like code completion and error checking.

The thing is, I use [pyenv](https://github.com/pyenv/pyenv) for managing different Python versions. Here's a screenshot of the installed versions:

<img width="912" alt="Screen Shot 2022-10-17 at 12 33 48 PM" src="https://user-images.githubusercontent.com/67194380/196105218-84355cd9-33c6-4faa-99d4-06184b061d9c.png">

Since Helix requires me to install `pylsp` with pip, should I opt for the system Python (3.9.6) or the one installed separately (3.10.7) by me?

**Comment by kirawi** [comment, marked as answer]
Helix will use whatever version of the LSP is on the `$PATH`. You can alternatively specify a file path to the executable.
